### DISCUSSION TITLE
Repo subscription clarification

### DISCUSSION THREAD

**Discussion by jsumners** [community/community#24654] [Apps, API and Webhooks]
<aside class="onebox allowlistedgeneric" data-onebox-src="https://docs.github.com/en/rest/reference/activity#set-a-repository-subscription">
  <header class="source">
      <img src="https://docs.github.com/assets/images/site/favicon.svg" class="site-icon" width="16" height="16">

      <a href="https://docs.github.com/en/rest/reference/activity#set-a-repository-subscription" target="_blank" rel="noopener nofollow ugc">GitHub Docs</a>
  </header>

  <article class="onebox-body">
    <img src="https://user-images.githubusercontent.com/321201/181085856-52aa3bd7-4a57-450a-8c4e-c6a22bacf293.png" class="thumbnail onebox-avatar" width="500" height="500" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/3/337b7cc06184984e148c075df20ef009e0e6ecaa_2_500x500.png, https://aws1.discourse-cdn.com/github/optimized/2X/3/337b7cc06184984e148c075df20ef009e0e6ecaa_2_750x750.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/2X/3/337b7cc06184984e148c075df20ef009e0e6ecaa_2_1000x1000.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/3/337b7cc06184984e148c075df20ef009e0e6ecaa_2_10x10.png">

<h3><a href="https://docs.github.com/en/rest/reference/activity#set-a-repository-subscription" target="_blank" rel="noopener nofollow ugc">Activity - GitHub Docs</a></h3>

  <p>The Activity API allows you to list events and feeds and manage notifications, starring, and watching for the authenticated user.</p>

  </article>

  <div class="onebox-metadata">
    
    
  </div>

  <div style="clear: both"></div>
</aside>

<p>I’m not clear on the documentation. Does <code>subscribed: false</code> + <code>ignored: false</code> translate to "Particpating and <code>@mentions</code>" when changing a subscription from the UI?</p>
<p><img src="https://user-images.githubusercontent.com/321201/181085858-33523d4f-3020-4ee1-add1-c876f97bf16e.png" alt="image" data-base62-sha1="qRfJcRTZGHiOhvWMFTfFzOkJLt0" width="386" height="470"></p>

**Comment by jsumners** [comment]
<p>:thinking: sending <code>{ subscription: false }</code> does nothing. Sending <code>{ ignore: true }</code> sets the repository subscription to “Never be notified”. I cannot find anything in the API that lets me set it to “Participating and mentions”.</p>

**Comment by jsumners** [comment, marked as answer]
<p>The solution is to send a <code>DELETE</code> request – <a href="https://docs.github.com/en/rest/reference/activity#delete-a-repository-subscription" rel="noopener nofollow ugc">https://docs.github.com/en/rest/reference/activity#delete-a-repository-subscription</a></p>
<p>Sending a <code>DELETE</code> puts your subscription status into “participating and mentions”. The documentation is not clear about this. My interpretation of the docs is that this would set <code>ignore: true</code>.</p>

**Comment by jfullstackdev** [comment]
<p>nice solution, I was about to comment, but I just waited for your update,</p>
